src/Util/System.hs

```diff
@@ -76,10 +76,13 @@ withTempdir subdir callback
        return result
 
 rmFile :: FilePath -> IO ()
-rmFile f = do putStrLn $ "Removing " ++ f
-              catchIO (removeFile f)
-                      (\ioerr -> putStrLn $ "WARNING: Cannot remove file "
-                                 ++ f ++ ", Error msg:" ++ show ioerr)
+rmFile f = do
+  fileExists <- doesFileExist f
+  when fileExists
+   $ do putStrLn $ "Removing " ++ f
```

Comment:
This dollar should really be on the previous line to fit in with the rest of the codebase. Something like:
```
when fileExists $
  do putStrLn $ "Removing" ++ f
     ...
```